lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -1667,6 +1671,25 @@ MDagPath PrimUpdaterManager::setupPullParent(const Ufe::Path& pulledPath, VtDict
     return pullParentPath;
 }
 
+void PrimUpdaterManager::recordPullVariantInfo(
+    const Ufe::Path& pulledPath,
+    const MDagPath&  pullParentPath)
+{
+    // Add the pull parent to our pulled prims prefix tree.  Also add the full
+    // configuration of variant set selections for each ancestor, up to the USD
+    // pseudo-root.  Variants on the pulled path itself are ignored, as once
+    // pulled into Maya they cannot be changed.
+    TF_VERIFY(!_pulledPrims.containsDescendantInclusive(pulledPath));
+    TF_VERIFY(pulledPath.runTimeId() == MayaUsd::ufe::getUsdRunTimeId());
```

Comment:
Again, curious: how could we possibly be pulling a node that is not a USD node? Just paranoia? :)